ihmc-robotics-toolkit/src/test/java/us/ihmc/robotics/geometry/ConvexPolygonToolsTest.java

```diff
@@ -378,6 +377,28 @@ private void plotPolygon(FrameConvexPolygon2D polygonWithLimitedVertices, int de
       testFrame.dispose();
    }
 
+   private void plotPolygons(ConvexPolygon2D polygonA, ConvexPolygon2D polygonB)
```

Comment:
This was used to provide the following visualization as an insight to fixing this test.
![image](https://github.com/user-attachments/assets/67e31add-f83d-4204-b152-818c1e1fca52)